man/loader.conf.xml

```diff
@@ -215,6 +215,18 @@
         Currently, only x86 is supported, where it uses the PC speaker.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>auto-enroll-keys</term>
+
+        <listitem><para>Takes a boolean argument. Enable or disable (the default) the auto enrollement
+        of secure boot keys. If this is enabled, the UEFI firmware is in setup-mode and a directory named
+        'auto' exists on the ESP under <filename>/loader/keys</filename> then the db, KEK and PK secure boot
```

Comment:
we usually suffix paths that refer to directories with a `/` in our docs. also, why not explicitly say `/loader/keys/auto/`?